quinn-proto/src/endpoint.rs

```diff
@@ -45,6 +45,8 @@ pub struct Endpoint {
     config: Arc<EndpointConfig>,
     server_config: Option<Arc<ServerConfig>>,
     incoming_handshakes: usize,
+    /// Whether incoming connections should be unconditionally rejected by a server
+    busy: bool,
```

Comment:
Don't like the name much, maybe `reject_new` or `reject_new_connections` or `draining`?